pkgs/top-level/all-packages.nix

```diff
@@ -26346,7 +26346,9 @@ in
 
   bcompare = libsForQt5.callPackage ../applications/version-management/bcompare {};
 
-  xp-pen-g430 = libsForQt5.callPackage ../misc/drivers/xp-pen-g430 {};
+  pentablet-driver = libsForQt5.callPackage ../misc/drivers/pentablet-driver { };
+
+  xp-pen-g430 = pentablet-driver; # Renamed on 2020-05-03
```

Comment:
This line belongs to `pkgs/top-level/aliases.nix`, not `pkgs/top-level/all-packages.nix`.

Reply:
Another option is to add there the following:

```
xp-pen-g430 = throw "xp-pen-g430 was renamed to pentablet-driver"; # added 2020-05-03
```

Reply:
Oh, I'm sorry. Misread your review.. Fixed.